localstack_services.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+
+basedir=$(pwd)
+
+
+printf "Configuring localstack components via terraform...\n"
+
+printf "=> Cleaning up previous caches and lambda dependency packages\n"
+
+printf "...Purging stale localstack related files\n"
+  find $basedir/env/local -type d -name .terragrunt-cache -prune -exec rm -rf {} \;
```

Comment:
Running this script I see a bunch of errors about terraform state. If I try running just the above command the terraform lock and tfstate files aren't removed. We could just have a simple delete for each related file?

Reply:
The terraform lock files shouldn't be deleted.  They're like yarn lock files for the versions of the terraform packages.
Let's connect.  We'll reset your env to brand new and try again.